src/store/domains/globalSearch.ts

```diff
@@ -53,6 +55,10 @@ export default class GlobalSearch implements GlobalSearchStoreInterface {
     return this.results;
   }
 
+  get getEntityType() {
```

Comment:
`getEntityType` wäre sinnvoll, wenn man einen von `entityType` abgeleiteten Wert zurückgeben wollen würde.

In Komponenten kann über den StoreContext, genauer der GlobalSearch-Store-Instanz auch direkt über `globalSearch?.filters.entityType`  auf `entityType` zugreifen.

Außer man möchte so einen "Getter" als Schnittstelle einbringen.